src/cli.ts

```diff
@@ -21,7 +21,16 @@ export function run(): Promise<void> {
       type: Boolean,
     });
     var cli = commandLineArgs(argsWithHelp);
-    var cliOptions = cli.parse();
+
+    try {
+      var cliOptions = cli.parse();
```

Comment:
what case was causing `parse` to throw? I'm seeing that the parsing is rather lenient?